Neos.Cache/Classes/Backend/PdoBackend.php

```diff
@@ -386,8 +385,12 @@ protected function connect()
     protected function createCacheTables()
     {
         $this->connect();
+        $sqlPath = __DIR__ . '/../../Resources/Private/DDL_' . strtolower($this->pdoDriver) . '.sql';
+        if (!file_exists($sqlPath)) {
+            throw new Exception(sprintf('Could not find SQL for pdo driver "%s" (at "%s")', $this->pdoDriver, $sqlPath), 1568133488);
+        }
         try {
-            PdoHelper::importSql($this->databaseHandle, $this->pdoDriver, __DIR__ . '/../../Resources/Private/DDL.sql');
+            $this->databaseHandle->exec(file_get_contents($sqlPath));
```

Comment:
We could first read the sql and then replace the table names. This way the method works with any number of configured caches.